client/i18n/locales/english/translations.json

```diff
@@ -593,16 +595,16 @@
     "add-code-three": "because they allow your code to properly format in the post."
   },
   "webhook-token": {
-    "copy-text": "Copy your user token below to save your progress in this section. There will be a place for you to paste it after a course loads.",
-    "copy": "Copy",
-    "generate": "Generate New Token",
-    "generate-p1": "If you suspect your token has been compromised, generate a new one.",
-    "generate-p2": "You will not be able to save progress in this section using your old token.",
-    "generate-confirm": "Yes please, I would like to generate a new token",
-    "revoke": "Revoke Token",
-    "revoke-p1": "If you suspect your token has been compromised, revoke it to make it unusable.",
-    "revoke-p2": "You will need to generate a new one to save progress in this section.",
-    "revoke-confirm": "Yes please, I would like to revoke my token",
-    "no-thanks": "No thanks, I would like to keep my token"
+    "title": "Webhook Token",
+    "create": "Create a new token",
+    "create-p1": "It looks like you don't have a webhook token. Create one to save your progress on this section",
+    "create-p2": "Create a webhook token to save your progress on curriculum that uses a virtual machine.",
+    "delete": "Delete my token",
+    "delete-title": "Delete My Webhook Token",
+    "delete-p1": "Your webhook token below is used to save your progress on curriculum that uses a virtual machine.",
+    "delete-p2": "If you suspect your token has been compromised, you can delete it to make it unusable. Progress on previously submitted lessons will not be lost.",
+    "delete-p3": "You will need to create a new token to save future progress on curriulum that uses a virtual machine.",
```

Comment:
```suggestion
    "create-p1": "It looks like you don't have a webhook token. Create one to save your progress on this section",
    "create-p2": "Create a webhook token to save your progress on the curriculum sections that use a virtual machine.",
    "delete": "Delete my token",
    "delete-title": "Delete My Webhook Token",
    "delete-p1": "Your webhook token below is used to save your progress on the curriculum sections that use a virtual machine.",
    "delete-p2": "If you suspect your token has been compromised, you can delete it to make it unusable. Progress on previously submitted lessons will not be lost.",
    "delete-p3": "You will need to create a new token to save future progress on the curriculum sections that use a virtual machine.",
```

Reply:
Just to clarify, this change doesn't need to be added to the rest of the languages, correct?